modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -485,7 +485,7 @@
 	glass_desc = "Warm milk and some catnip."
 
 /datum/reagent/consumable/ethanol/frisky_kitty/on_mob_life(mob/living/carbon/M)
-	if(ismammal(M) || isfelinid(M) || istajaran(M))
+	if(isfeline(M))
```

Comment:
Considering how it's not a disgusting drink to non-Felinids, I'm not entirely sure why this is needed here.

Reply:
I'm changing this in another PR so I've not put too much thought into this one here